projecttacoma/fhir-spec-tools
# Initial scripts

# Summary
Add all fhir spec manipulation scripts to this shared repository.

## New behavior
Behavior should remain the same

## Code changes
- Makes small changes to script/file naming to have some consistency while still remaining obvious where the script came from
- Converts some scripts to typescript
- Create package.json shortcut for all scripts
- Adds new parseDateSearchParam.ts script (there may have been small updates on the R4 date search parameters as there are now 109 parameters versus the previous 106 parameters in the file currently in fqm-execution)
- Add description of all scripts to the README

# Testing guidance

- Read through README
- Run all scripts from package.json shortcuts
- Ensure that output files match up as described in README
